tests/system/testdata/authz/default/connections.txtar

```diff
@@ -0,0 +1,53 @@
+user zumi
+
+ak project create --name pz
+return code == 0
+capture_jq zpid .project_id
+
+ak project create --name pc --org cats
+return code == 0
+capture_jq cpid .project_id
+
+ak connection create c1 --project $zpid --integration test
+return code == 0
+capture_jq cid .connection_id
+
+ak connection list --project $zpid
+return code == 0
+
+user gizmo
+
+ak connection create c2 --project $cpid --integration test
+return code == 0
+capture_jq ccid .connection_id
+
+ak connection create nope --project $zpid --integration test
+return code == $RC_UNAUTHZ
+
+ak connection list --project $cpid
+return code == 0
+
+ak connection list --project $zpid
+return code == $RC_UNAUTHZ
```

Comment:
Shouldn't list for someone else's stuff also return NOT FOUND instead of UNAUTHZ?